forc/src/cli/commands/build.rs

```diff
@@ -7,12 +7,18 @@ pub struct Command {
     /// Path to the project, if not specified, current working directory will be used.
     #[structopt(short, long)]
     pub path: Option<String>,
+    /// Whether to compile using the IR pipeline.
+    #[structopt(long)]
+    pub use_ir: bool,
     /// Whether to compile to bytecode (false) or to print out the generated ASM (true).
     #[structopt(long)]
     pub print_finalized_asm: bool,
     /// Whether to compile to bytecode (false) or to print out the generated ASM (true).
     #[structopt(long)]
     pub print_intermediate_asm: bool,
+    /// Whether to compile to bytecode (false) or to print out the generated IR (true).
+    #[structopt(long)]
+    pub print_ir: bool,
```

Comment:
Comment seems wrong. Aside: these flags that are dependent on each other remind me of https://www.youtube.com/watch?v=bnnacleqg6k.